mono/utils/mono-time.c

```diff
@@ -150,8 +150,8 @@ mono_msec_boottime (void)
 {
 	gint64 retval = 0;
 
-	/* clock_gettime () is found by configure but its only present from ios 10 */
-#if (defined(HAVE_CLOCK_MONOTONIC_COARSE) || defined(HAVE_CLOCK_MONOTONIC)) && !defined(TARGET_IOS)
+	/* clock_gettime () is found by configure but its only present from ios 10 and macos 10.12 */
+#if (defined(HAVE_CLOCK_MONOTONIC_COARSE) || defined(HAVE_CLOCK_MONOTONIC)) && !(defined(TARGET_IOS) || defined(TARGET_OSX))
```

Comment:
What about TARGET_WATCHOS/TARGET_TVOS?

Reply:
I was planning letting us hit the problem only if it exists since the fix is trivial, I had no idea if they had they had the symbol because they are newer and e.g. mach_absolute_time is not available. But now I found this: https://github.com/Haivision/srt/blob/master/common/srt_compat.h#L45

So I'll do a touchup, thanks :)